### ISSUE TITLE
Issue with Asserts?

### ISSUE THREAD

**Issue by karpikpl** [Avanade/UnitTestEx#51]
I have a following tests that passes with `"UnitTestEx.Xunit" Version="2.1.1"`
```csharp
		// Arrange
		using var test = CreateApiTester<Startup>();

		// Act
		var result = test
			.Controller<SampleController>()
			.Run(c => c.Get("thisIsATest"))
			.AssertOK()
			.AssertJson("{\"data\": \"this_is_a_test\"}");
```

but this also passes and shouldn't
```csharp
		// Act
		var result = test
			.Controller<SampleController>()
			.Run(c => c.Get("thisIsATest"))
			.AssertOK()
			.Assert(new { foo="bar"});
```

**Comment by chullybun** [comment]
Published NuGet package v2.1.2.

**Comment by karpikpl** [comment]
fix worked
